docs/nodes/validate/how-to-stake.md

```diff
@@ -78,6 +78,7 @@ probability that a given validator is sampled is proportional to its stake.
 When you add a node to the validator set, you specify:
 
 - Your node’s ID
+- Your node's BLS key and BLS signature
 - When you want to start and stop validating
 - How many AVAX you are staking
```

Comment:
you can also add a comment on this file, regarding backing up the bls key file, on `Secret Management` section